Makefile

```diff
@@ -5,6 +5,12 @@ VERSION := $(shell sed -n -e 's/version:[ "]*\([^"]*\).*/\1/p' plugin.yaml)
 DIST := $(CURDIR)/_dist
 LDFLAGS := "-X main.version=${VERSION}"
 
+PHONY: modules
+modules:
+	export GOPROXY=https://gocenter.io && \
+	export GO111MODULE=on && \
+	go mod download
+
```

Comment:
How about writing it something like this (might need to rework the syntax):

```
MOD_PROXY_URL ?= https://gocenter.io

.PHONY: bootstrap
bootstrap: 
                export GO111MODULE=on && \
                export GOPROXY=$(MOD_PROXY_URL) && \
	        go mod download
```

Reply:
sure

Reply:
done